features/cron.feature

```diff
@@ -360,3 +364,29 @@ Feature: Manage WP-Cron events and schedules
     Then STDOUT should be CSV containing:
       | hook                   | recurrence    | args                           |
       | wp_cli_test_args_event | Non-repeating | {"foo":"banana","bar":"apple"} |
+
+Scenario: Delete all cron events
+    When I try `wp cron event delete`
+    Then STDERR should be:
+      """
+      Error: Please specify one or more cron events, or use --due-now/--all.
+      """
+
+    # WP throws a notice here for older versions of core.
+    When I try `wp cron event delete --all`
+    Then STDOUT should contain:
+      """
+      Deleted the cron event 'wp_version_check'
+      """
+    And STDOUT should contain:
+      """
+      Deleted the cron event 'wp_update_plugins'
+      """
+    And STDOUT should contain:
+      """
+      Deleted the cron event 'wp_update_themes'
+      """
+    And STDOUT should contain:
+      """
+      Success: Deleted a total of
+      """
```

Comment:
Are these events 100% save to rely on? I get they get registered every new install, but what if core changes, or other tests in the future.
Might be safer to add a couple of custom events [like here](https://github.com/wp-cli/cron-command/blob/master/features/cron.feature#L77).

Reply:
I've copied this test from the 'Run multiple cron tests' scenario on line 230. I'll update that test as well.